src/test/java/com/neuvector/ScannerTest.java

```diff
@@ -67,7 +67,35 @@ public void scanRegistryTest() throws IOException {
         String mountPath = mountFolder.getRoot().getAbsolutePath();
 
         Registry registry = new Registry(registryURL, regUser, regPassword,repository,repositoryTag);
-        NVScanner scanner = new NVScanner(nvScannerImage, nvRegistryURL, nvRegistryUser, nvRegistryPassword, mountPath, null);
+        NVScanner scanner = new NVScanner(nvScannerImage, nvRegistryURL, nvRegistryUser, nvRegistryPassword, mountPath, null, null);
+
+        ScanRepoReportData scanReportData = Scanner.scanRegistry(registry, scanner, license);
+
+        UserPrincipal user = getUserPrincipal(mountPath);
+
+        assertFalse(user.getName().equals("root"));
+        assertTrue( scanReportData != null );
+    }
+
+    @Test
+    public void scanRegistryTestAddingSELinuxSuffixToMountPath() throws IOException {
```

Comment:
Method names need to be updated too.. Anything that references `SELinux`.. In both repositories.

Reply:
Commit: [a55ee47](https://github.com/neuvector/scanner-java-wrapper/pull/11/commits/a55ee4758cc3ed7dbc9abf39326de51df7a62917)